include/zephyr/pm/policy.h

```diff
@@ -196,10 +202,12 @@ void pm_policy_latency_changed_unsubscribe(struct pm_policy_latency_subscription
  *
  * @param evt Event.
  * @param time_us When the event will occur, in microseconds from now.
+ * @param low_latency If True indicates that the system needs to be up before the
+ *                    events happens
  *
  * @see pm_policy_event_unregister
  */
-void pm_policy_event_register(struct pm_policy_event *evt, uint32_t time_us);
+void pm_policy_event_register(struct pm_policy_event *evt, uint32_t time_us, bool low_latency);
```

Comment:
low latency events seem to be a configurable option, but this API is always available, so it will change its behavior depending on a switch, not sure if this is a good pattern...

Reply:
I would at least add information to `@param low_latency` that this parameter is ignored if `CONFIG_...`

Reply:
@ceolin we need your advice here.
What is the best solution in this case ? make two functions (one with extra argument) and protect them using KCONFIG or keep the extra argument and ignore it when the KCONFIG option is not selected ?

Reply:
Another option would be to create a new function like:

```
#ifdef CONFIG_PM_LOW_LATENCY_EVENTS
void pm_policy_low_latency_event_register(struct pm_policy_event *evt, uint32_t time_us);
#endif
```